checker/jdk/index/src/java/io/File.java

```diff
@@ -195,7 +195,7 @@ int getPrefixLength() {
      * string for convenience.  This string contains a single character, namely
      * <code>{@link #separatorChar}</code>.
      */
-    public static final @MinLen(1) String separator = "" + separatorChar;
+    public static final /*!MinLen(1)*/ String separator = "" + separatorChar;
```

Comment:
I don't think any Value Checker annotations should be in the Index jdk.  (Including ones that are commented out.)